.github/workflows/docker.yml

```diff
@@ -11,20 +11,37 @@ on:
       - "verification/curator-service/**"
 
 jobs:
-  test:
+  test-dev:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Build Stack
+      - name: Build stack
         run: docker-compose -f dev/docker-compose.yml -f dev/docker-compose.ci.yml up --build -d -V
       # Containers are run detached above, which means we have to wait until they are up to produce meaningful logs.
       - name: Sleep for 30 seconds
         uses: jakejarvis/wait-action@master
         with:
           time: "30s"
-      - name: Print Logs
+      - name: Print logs
         run: docker-compose -f dev/docker-compose.yml logs
-      - name: Test Curator Service
+      - name: Fetch Curator Service sources
         run: docker-compose -f dev/docker-compose.yml exec -T curator curl --retry-connrefused --retry 20 --retry-delay 5 -f --silent --show-error http://localhost:3001/api/sources
-      - name: Test Data Service
+      - name: Test Data Service cases
         run: docker-compose -f dev/docker-compose.yml exec -T data curl --retry-connrefused --retry 20 --retry-delay 5 -f --silent --show-error http://localhost:3000/api/cases
+  test-prod:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build stack
+        run: docker-compose -f prod/docker-compose.yml -f prod/docker-compose.ci.yml up --build -d -V
+      # Containers are run detached above, which means we have to wait until they are up to produce meaningful logs.
+      - name: Sleep for 30 seconds
+        uses: jakejarvis/wait-action@master
+        with:
+          time: "30s"
+      - name: Print logs
+        run: docker-compose -f prod/docker-compose.yml logs
+      - name: Fetch Curator Service index page
```

Comment:
Is this the best "test" we can get in here? Seems like what we'd want to fail on from a docker action is a non-zero build code. I'm surprised this is our pass/fail condition.

Reply:
A failed build step will trigger a whole workflow failure so we're testing that already actually, my plan is to remove those curl tests once we have a proper "integration test" that's hopefully coming this sprint :)